tools/cut_a_release.md

```diff
@@ -0,0 +1,47 @@
+# Cutting a Deno release
+
+**During this process `main` branch should be frozen and no commits should land
+until the release is cut.**
```

Comment:
Are there differences to account for versus creating a main release versus a branch patch release?

Reply:
Added